Dockerfile.swarm

```
@@ -21,9 +21,10 @@ ENV PUBLIC_URL=$public_url
 RUN yarn build:$environment
 
 FROM nginx:1.19.0-alpine AS final
+ARG environment
```

Should we add the default value?

```suggestion
ARG environment=production
```